[DLStreamer] Add guide for opencv removal (#514)
open-edge-platform/edge-ai-libraries: libraries/dl-streamer/docs/source/dev_guide/advanced_install/advanced_install_guide_compilation.rst

    ninja -C build
    sudo env PATH=~/python3venv/bin:$PATH meson install -C build/

Step 6: Build OpenCV
^^^^^^^^^^^^^^^^^^^^

NOTE: If you have built and installed a different version of OpenCV locally, it can cause build errors. It is recommended to uninstall it first.
You can uninstall it with the following command (if installed from source):

.. code:: sh

    cd ${HOME}/opencv/build # Change to the directory where OpenCV was built
    sudo ninja uninstall
        
.. tabs::

    .. tab:: Ubuntu 24

        After uninstalling OpenCV, reinstall it with the following command:
        
        .. code:: sh

            sudo apt-get install --reinstall libopencv-dev

    .. tab:: Ubuntu 22
        

>         NOTE: If you have installed different version of OpenCV using apt-get, it is recommended to uninstall it first.
>         You can uninstall it with the command below:

Reviewer: ```suggestion
        NOTE: If you have installed different version of OpenCV using apt-get, you can uninstall it with the command below instead:
```

    ninja -C build
    sudo env PATH=~/python3venv/bin:$PATH meson install -C build/

Step 6: Build OpenCV
^^^^^^^^^^^^^^^^^^^^

> NOTE: If you have build and installed different version of OpenCV locally it can cause build error and it is recomended to uninstall it first.

Reviewer: ```suggestion
NOTE: If you have built and installed a different version of OpenCV locally, it can cause build errors. It is recommended to uninstall it first.
```

> If you have installed OpenCV from source you can uninstall it with the following command:

Reviewer: ```suggestion
You can uninstall it with the following command (if installed from source):
```

.. code:: sh

    cd ${HOME}/opencv/build # Change to the directory where OpenCV was built
    sudo ninja uninstall
        
.. tabs::

    .. tab:: Ubuntu 24

>         NOTE: If you have build and installed different version of OpenCV locally it can cause build error and it is recomended to uninstall it first.
>         You can uninstall it with the command above.
>         And then reinstall OpenCV with the following command:

Reviewer: ```suggestion
        After uninstalling OpenCV, reinstall it with the following command:
```
